projects/angular/src/forms/password/password-container.ts

```diff
@@ -41,6 +41,7 @@ export const TOGGLE_SERVICE_PROVIDER = { provide: TOGGLE_SERVICE, useFactory: To
             <cds-icon status="info" class="clr-password-eye-icon" [attr.shape]="show ? 'eye-hide' : 'eye'"></cds-icon>
             <span class="clr-sr-only">
               {{ show ? commonStrings.keys.passwordHide : commonStrings.keys.passwordShow }}
+              {{ commonStrings.keys.passwordFor + ' ' + label?.labelText }}
```

Comment:
Common strings cannot concatenated like this because languages have different words order. In other to translate properly, we use placeholders in the string. See the `comboboxSearching` string for an example.

Reply:
@kevinbuhmann Made the suggested changes and updated the PR.

Reply:
Thanks, approved!